src/component/panels/SummaryPanel/Utilities.ts

```diff
@@ -13,6 +13,7 @@ function getLabelColor(correlationData, correlation) {
     null,
   );
   if (error) {
+    // eslint-disable-next-line @typescript-eslint/no-for-in-array
```

Comment:
there is a mistake in the code. It should iterate directly on `ErrorColors` with for of